examples/simpack_xrd_dyn_AlGaAs.py

```diff
@@ -0,0 +1,178 @@
+# This file is part of xrayutilities.
+#
+# xrayutilities is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Copyright (c) 2016-2023 Dominik Kriegner <dominik.kriegner@gmail.com>
+
+
+
+# This is an example of the modelling and fitting of the diffraction data obtained on epitaxial heterostructure
+# The design is as follow
+#               _________________________
+#
+#                       GaAs 320 nm          using the same Ga cell as GaAs below (same growth rate)
+#               _________________________
+#
+#                  Al_0.80GaAs 1.8 micron    using the same Ga & Al cells as Al_0.80GaAs below (same growth rate and composition)
+#               _________________________
+#
+#                       GaAs 200 nm           
+#               _________________________
+#
+#                  Al_0.80GaAs 2 micron
+#               _________________________
+#
+#              GaAs substrate and buffer (001)
+#
+# Xray set-up: X'PERT Panalytical with
+#       copper anode,focusing mirror, Ge(220)x4 monochromator, Ge(220)x2 triple axis detection
+# Copyright (c) 2023 C2N - CNRS Author:	  Aristide Lemaître <aristide.lemaitre@c2n.upsaclay.fr>
+
+
+
+import xrayutilities as xu
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def x_Al(aper,apar,mat):
+    # determination of Al composition from c parameter, poisson coefficient and Veegard's law
+    c11 = mat.cij[0,0] # approx of C11 and C12 from alloy with 
+    c12 = mat.cij[0,1]
+    eps_star = (apar - aper)/apar
+    eps_AlAsGaAs = (AlAs_.a - GaAs_.a)/GaAs_.a
+    x = -c11 * eps_star / eps_AlAsGaAs / (c11 * (1 + eps_star) + 2 * c12) 
+    return x
+
+
+# read experimental file
+
+file ='data/simpack_xrd_AlGaAs.xrdml'
```

Comment:
file is only used in the next line. It therefore does not need to be defined as variable and the filename can directly go in the next line. Also to keep better cross-platform compatibility use:
`scan = xu.io.panalytical_xml.XRDMLFile('simpack_xrd_AlGaAs.xrdml', path='data')`

Reply:
done